src/core/js/helpers.js

```diff
@@ -362,6 +362,7 @@ define([
          * @param {string} visible Text that will be displayed on screen
          * @param {string} alternatives Text that will be read by the screen reader (instead of what's displayed on screen)
          * @usage {{a11y_alt_text '$5bn' 'five billion dollars'}}
+         * @usage {{a11y_alt_text '$5bn' 'five' 'billion' 'dollars'}}
```

Comment:
sorry but I meant the more recent example, also you don't have to put two @usage entries in you can just do it like this:
> @usage `{{a11y_alt_text '$5bn' 'five billion dollars'}}` or `{{a11y_alt_text 'Here are some bits to read' 'There are' _items.length 'items to read'}}`